src/components/Calculator/Calculator.spec.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Calculator from './Calculator';
+
+describe('Calculator', () => {
+    let wrapper;
+    beforeEach(() => wrapper = shallow(<Calculator />))
+    it('should render a div', () => {
+        expect(wrapper.find('div').length).toEqual(1);
+    });
+
+    it('should have the class for Calculator', () => {
```

Comment:
On EXH, we've chosen to remove `should` from our it blocks to show confidence in our tests. It also reads better grammatically once you begin to become more descriptive in your conditionals

```suggestion
    it('contains a div with className "calculator-container"', () => {
```